storage/kafka/datasource_test.go

```diff
@@ -0,0 +1,164 @@
+package kafka
```

Comment:
Isn't there anything else to test for this data source? I know it doesn't support any keys, but two test cases seems like a small amount :P

Reply:
No, not really.